docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1647 @@
+{
```

Comment:
<p>As mentioned, if we collate the individual runs then we can use a box-and-whiskers or violin plot to show all of this data in one, I think.</p><p> </p><p>There's also a question of what the fairest comparison is. I think, if possible, we should also show the "first run" cost if loading the jitted code from the disk cache (I think that's a thing?). Since you might run the same workflow many times from scratch, but if the cache persists that's the number you care about.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>